bin/sql-formatter

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env php
 <?php
 
-if("cli" !== php_sapi_name()) {
+use Doctrine\SqlFormatter\SqlFormatter;
+
+if('cli' !== php_sapi_name()) {
```

Comment:
This kind of CS change should be handled by PHPCS. Can we please configure PHPCS so that it runs on the files you've changed here?

Reply:
@greg0ire you are an CI expect, might I know why the phpcs is not working for examples/ and for bin/?

examples/ - they are added in https://github.com/doctrine/sql-formatter/blob/e1e39d8f4c/phpcs.xml.dist#L22 and ending with `.php`, but it seems they are completely ignored as for ex. constants like in https://github.com/doctrine/sql-formatter/blob/e1e39d8f4c/examples/cli.php#L10 are not added to uses.

bin/ - the file does not end with `.php` but a full path is specified - it seems ignored currently as well as I would expect the `php_sapi_name()` in https://github.com/doctrine/sql-formatter/blob/e1e39d8f4c/bin/sql-formatter#L6 to be replaced with a constant.

Reply:
It works, it's just that you did not spot https://github.com/doctrine/sql-formatter/blob/7c0096a099fe3480b742780ba2091856944d3cbc/phpcs.xml.dist#L25-L36

Reply:
I spot the config but found `UnusedUses` rule not relevant...

So in `examples/` - `InlineHTML` is still preventing any CS rules when a single inline HTML/PHP is found?

And in `bin/` the CS rules are still not applied.

Reply:
Well in `bin`, there are no files ending with `.php` :man_shrugging:

Reply:
> `InlineHTML` is still preventing any CS rules when a single inline HTML/PHP is found?


You don't need a working Github CI to figure that out, do you?

Reply:
Consider filing an issue at https://github.com/PHPCSStandards/PHP_CodeSniffer/